tests/installation/hostname_inst.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
Please only keep the years where you worked on this new file, I guess only 2016 :-)